KUBEVIRT_LOCAL_TESTING.md

```diff
@@ -1,34 +1,52 @@
 # [kubevirtci](README.md): Testing kubevirt with locally provisioned cluster
 
-With the changes in place you also can directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+After making changes to a kubevirtci provider, you should test it locally before publishing it.
+
+With the changes in place you can also directly execute [`make functest`](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md#testing) against a cluster locally with kubevirt that was provisioned using `kubevirtci`.
+
+Lets go on the required steps, starting by provisioning, all the way to `make functest`.
 
 Steps:
 
 ### kubevirtci: provision cluster locally
 
 ```bash
 cd .../kubevirtci
-cd cluster-provision/k8s/1.17.0
+cd cluster-provision/k8s/1.19
 ../provision.sh      # this also calls cluster-up to check whether the cluster will really start and have the pods ready
 ```
 
 ### kubevirt: prepare for tests
 
 ```bash
-# set local provision test flag
+# set local provision test flag (mandatory)
 export KUBEVIRTCI_PROVISION_CHECK=1
-# sync changes to kubevirt cluster-up, notably cluster/images.sh
+# sync changes to kubevirt cluster-up, notably cluster/images.sh (needed in case kubevirt testing is needed as well)
```

Comment:
I don't understand the comment, nor your added parenthesis.
I don't even understand what are the `...` on the `rsync` commandline.

Reply:
the `...` is the notation in this file to say "whatever the path before that"
so `rsync -av .../kubevirtci/cluster-up .../kubevirt/cluster-up`
means rsync between your kubevirtci/cluster-up folder to your kubevirt/cluster-up folder,
whatever those two paths parent dir is (this is the `...`)

i will update the whole line, because some of it is outdated (images.sh is deprecated)
and the _ci-configs/ is the thing that need to be synced in case kubevirt should be tested.